lib/ably/realtime/connection.rb

```diff
@@ -439,9 +439,22 @@ def send_protocol_message(protocol_message)
         end
       end
 
+      def send_protocol_message_immediately(protocol_message)
+        Ably::Models::ProtocolMessage.new(protocol_message, logger: logger).tap do |message|
```

Comment:
Why `#tap`? That's usually used when you want to create an object and also return it. Can you just do

```ruby
message = Ably::Models::ProtocolMessage.new(protocol_message, logger: logger)
add_message_to_outgoing_queue(message, true)
(and so on)
```

Reply:
Fixed https://github.com/ably/ably-ruby/pull/416/commits/7a973ae7718d2019544a61ddfe11e115cc0654ac